lib/chef/provider/google_key_pair.rb

```diff
@@ -41,20 +41,21 @@ class Chef::Provider::GoogleKeyPair < Chef::Provider::LWRPBase
   end
 
   action :destroy do
-    name_to_key = driver.project_client.get_ssh_mappings
+    metadata = driver.project_client.get
```

Comment:
@Lykos Have you thought of abstracting the metadata call too to avoid future confusion if we want to get any other info from the project?
As in 
`driver.project_client.get.metadata` `driver.project_client.get.metadata.ssh_mappings`
or, alternatively:
`driver.project_client.get.metadata` `driver.project_client.get.ssh_mappings`